soccer/src/soccer/strategy/agent/position/robot_factory_position.cpp

```diff
@@ -49,29 +50,29 @@ std::optional<RobotIntent> RobotFactoryPosition::get_task(WorldState& world_stat
     // Assigning new position
     // Checking whether we have possesion or if the ball is on their half (using 1.99 to avoid
     // rounding issues on midline)
-    if (Position::our_possession_ ||
-        world_state.ball.position.y() > field_dimensions.length() / 1.99) {
+    if (our_possession_ ||
+        world_state.ball.position.y() > field_dimensions.center_field_loc().y() - kBallDiameter) {
         // Offensive mode
         // Closest 2 robots on defense, rest on offense
         if (i <= 1) {
```

Comment:
While we're working on this file, could we change the i variable name to something more descriptive? Perhaps `robot_y_index`?
Variable defined on line 42 but it's collapsed so GitHub won't let me comment there